src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java

```diff
@@ -388,6 +389,12 @@ public void testCheck() throws Exception {
         verify(checkConfig, getPath("InputAbstractCheckTestFileContents.java"), expected);
     }
 
+    @Test
```

Comment:
Done.

Reply:
Please add some more details here, such as inspection is valid, but we cannot produce a case where x happens because….. so we use a pure unit test instead of suppressing inspection.

Reply:
Done.